tests/test_pgext.py

```diff
@@ -842,8 +842,9 @@ async def test_sql_proto_extended_query_08(self):
             Execute("portal8"),
             Sync(),
         )
-        await self.conn.read(ParseComplete)
-        await self.conn.read(BindComplete)
+        # On Postgres we will also receive:
```

Comment:
What does this comment mean exactly?

Reply:
This is to mark a behavioral difference between vanilla Postgres and our implementation

Reply:
Could it cause problems for clients?

Reply:
Theoretically yes, like clients sending `Parse` wouldn't expect a `42P03: cursor already exists` error; but it's an error after all, so I guess most clients would just raise the error regardlessly?

Reply:
OK, I guess we'll see, but ideally we shouldn't deviate too much from vanilla Postgres.